diff -urN evolution-rss-0.1.0/configure.ac evolution-rss-0.1.0.new/configure.ac
--- evolution-rss-0.1.0/configure.ac	2008-07-01 21:19:03.000000000 +0200
+++ evolution-rss-0.1.0.new/configure.ac	2008-07-09 19:45:56.101155584 +0200
@@ -19,7 +19,7 @@
 if test "x$GPP" = "xno"; then
 	AC_MSG_ERROR(C++ Compiler (g++) not installed)
 fi
-#AC_PROG_CXXCPP
+AC_PROG_CXX
 AC_ISC_POSIX
 AC_HEADER_STDC
 AM_DISABLE_STATIC
@@ -344,7 +344,7 @@
 serverdir="$BONOBO_LIBDIR/bonobo/servers"
 AC_SUBST(serverdir)
 
-privdatadir='${datadir}'/evolution/$EVOLUTION_EXEC_VERSION
+privdatadir='${datadir}'/evolution/"$EVOLUTION_EXEC_VERSION"
 AC_SUBST(privdatadir)
 
 gladedir="$privdatadir/glade"
